packages/framework/block-std/src/event/control/pointer.ts

```diff
@@ -268,9 +268,30 @@ class DragController extends PointerControllerBase {
     this._reset();
   };
 
+  // https://mikepk.com/2020/10/iOS-safari-scribble-bug/
+  private _applyScribblePatch() {
+    const isiPad =
```

Comment:
Should be put at https://github.com/toeverything/blocksuite/blob/master/packages/framework/global/src/env/index.ts

Reply:
Done